applications/acdc/src/acdc_announcements_sup.erl

```diff
@@ -0,0 +1,94 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, Voxter Communications
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Daniel Finke
+%%%-------------------------------------------------------------------
+-module(acdc_announcements_sup).
+
+-behaviour(supervisor).
+
+%% API
+-export([start_link/0]).
+-export([maybe_start_announcements/3
+        ,stop_announcements/1
+        ]).
+
+%% Supervisor callbacks
+-export([init/1]).
+
+-include("acdc.hrl").
+
+-define(SERVER, ?MODULE).
+-define(CHILDREN, [?WORKER('acdc_announcements')]).
+
+%%%===================================================================
+%%% API functions
+%%%===================================================================
+
+%%--------------------------------------------------------------------
+%% @doc
+%% Starts the supervisor
+%%
+%% @end
+%%--------------------------------------------------------------------
+-spec start_link() -> sup_startchild_ret().
+start_link() ->
+    supervisor:start_link({local, ?SERVER}, ?MODULE, []).
+
+%%--------------------------------------------------------------------
+%% @doc
+%% Start an announcements child if either position or wait time
+%% announcements are enabled
+%%
+%% @end
+%%--------------------------------------------------------------------
+-spec maybe_start_announcements(pid(), kapps_call:call(), kz_proplist()) -> supervisor:startchild_ret() | 'false'.
+maybe_start_announcements(Manager, Call, Props) ->
+    Enabled = props:get_is_true(<<"position_announcements_enabled">>, Props, 'false')
+        orelse props:get_is_true(<<"wait_time_announcements_enabled">>, Props, 'false'),
+    case Enabled of
```

Comment:
You can cheat and do:
```erlang
Enabled
    andalso supervisor:start_child(...)
```
`andalso` will not evaluate the right-hand side if the left is 'false' (and returns false).